metadata/contributors.json

```diff
@@ -4503,6 +4503,7 @@
    },
    {
       "emails" : [
+         "jmichaud@igalia.com",
```

Comment:
IIRC the first address needs to match Bugzilla account for CC autocomplete to work.